src/backend/efiling-api/jag-efiling-api.yaml

```diff
@@ -617,6 +617,31 @@ paths:
       responses:
         "200":
           description: Document was deleted
+  "/filingpackages/{packageIdentifier}/rushDocument/{documentIdentifier}":
+    get:
+      summary: Get rush document
+      operationId: GetRushDocument
+      description: get rush document
+      tags:
+        - filingPackages
+      parameters:
+        - $ref: "#/components/parameters/packageIdentifier"
+        - name: documentIdentifier
+          in: path
+          description: document identifier
+          required: true
+          schema:
+            type: string
+      responses:
```

Comment:
missing a 403 here I think

Reply:
It would be the only one. We only have 403 on client endpoints.